src/Slider.js

```diff
@@ -426,9 +450,11 @@ export default class Slider extends PureComponent<SliderProps, SliderState> {
 
   _thumbHitTest = (e: Object) => {
     const nativeEvent = e.nativeEvent;
-    this.changeThumSize(24);
     const thumbTouchRect = this._getThumbTouchRect();
-    this._bubble.press();
+    
+    this._setCurrentValue(Math.floor(100 * nativeEvent.locationX / this.state.containerSize.width));
```

Comment:
what 100 means? can you move it to constant with a meanigful name?

Reply:
100% of bar width